src/EditorFeatures/CSharpTest/OnTheFlyDocs/OnTheFlyDocsUtilitiesTests.cs

```diff
@@ -110,21 +88,620 @@ struct CustomStruct
             }
             """;
 
-        using var workspace = EditorTestWorkspace.CreateCSharp(testCode);
+        var customTypeText = """
+            class CustomClass
+            {
+                public string Name { get; set; }
+            }
+            """;
+
+        var customStructText = """
+            struct CustomStruct
+            {
+                public int Value { get; set; }
+            }
+            """;
+
+        await TestInvocationContextAsync(testCode, customTypeText, resultIndex: 0);
+        await TestInvocationContextAsync(testCode, customStructText, resultIndex: 1);
+    }
```

Comment:
the cleanup is very appreciated.  consider calling this like so:

```suggestion
    await TestInvocationContextAsync(testCode, customTypeText, customStructText);
```